app/routes/_landing.($lang).docs/components/footer/footer.tsx

```diff
@@ -22,31 +24,35 @@ const commonLinkStyles = (theme: MantineTheme) => ({
   color: "white",
   borderRadius: "0.5rem",
   "&:hover": {
-    backgroundColor: theme.colors.navy[4],
+    backgroundColor: "#192430",
     textDecoration: "none",
   },
 })
 
-function DocsFooter({ items }: DocsFooterProps) {
-  const theme = useMantineTheme()
-  const location = useLocation()
-  const lastPathPart = location.pathname.split("/").pop()
-  const [prev, next] = getNextAndPrevNodesInTree(
-    items,
-    items.find((ft) => lastPathPart === ft.slug),
+function DocsFooter({ items, pathname }: DocsFooterProps) {
+  const lastPathPart = pathname.split("/").pop()
+  const [prev, next] = useMemo(
+    () =>
+      getNextAndPrevNodesInTree(
+        items,
+        items.find((ft) => lastPathPart === ft.slug),
+      ),
+    [items, lastPathPart],
   )
 
+  let justify = "flex-end"
```

Comment:
This is one of the cases I'd love to use nested ternary. I know it produces a lint error currently... but yeah just saying.

Reply:
I was thinking of an easier way to do this... I think we can do it using css without needing the justify prop. 
We can use one of the two solutions.. 
1- Change the container display to `grid` and then use `justifySelf` properties to be `"flex-start"` for prev and "`flex-end"` for next.
2- Simply get rid of the `justify` const and use `marginRight: "auto"` for the prev doc link and `marginLeft: "auto"` for next doc link.

What do you think? @PseudoMera